src/glyph/mod.rs

```diff
@@ -125,6 +126,11 @@ impl Glyph {
         }
     }
 
+    /// Returns the name of the glyph.
+    pub fn name(&self) -> Name {
+        self.name.clone()
+    }
```

Comment:
Although this clone is not expensive, I think as a general pattern it's good to leave this decision up to the caller, unless we are returning these as cloned values in other places?

```suggestion
    pub fn name(&self) -> &Name {
        &self.name
    }
```